README.md

```diff
@@ -131,7 +131,13 @@ highlighting:
 git show v0.6.0:src/main.rs | bat -l rs
 ```
 
-Note that syntax highlighting within diffs is currently not supported. If you are looking for this, check out [`delta`](https://github.com/dandavison/delta).
```

Comment:
I really would love to keep the delta recommendation.

Reply:
Sounds good. I considered keeping it.